packages/autorest.go/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autorest/go",
-  "version": "4.0.0-preview.71",
+  "version": "4.0.0-preview.72",
```

Comment:
Changelog?

Reply:
Updated in https://github.com/Azure/autorest.go/pull/1530